recipes/azure-storage-blobs-cpp/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "azure-storage-blobs-cpp" %}
+{% set version = "12.7.0" %}
+{% set sha256 = "a28a665c740ae510053afa2c9b794052aa4de11d122b3ac4eaa0bb3bccb8983e" %}
+
+package:
+  name: {{ name|lower }}
```

Please don't use the name variables, and just set this directly
```suggestion
  name: azure-storage-blobs
```